.github/ISSUE_TEMPLATE/release.yaml

```diff
@@ -42,6 +42,14 @@ body:
       - label: Open a PR from your branch to `main`.
       - label: Once the PR is merged, [initiate a release job](https://github.com/Kong/kubernetes-ingress-controller/actions/workflows/release.yaml). Your tag must use `vX.Y.Z` format. Set `latest` to true if this will be the latest release.
       - label: CI will validate the requested version, build and push an image, and run tests against the image before finally creating a tag and publishing a release. If tests fail, CI will push the image but not the tag or release. Investigate the failure, correct it as needed, and start a new release job.
+- type: checkboxes
+  id: release_redhat
+  attributes:
+    label: "**For all releases** Publish images to Red Hat Catalog"
+    options:
+      - label: Verify the image was pushed by a pipeline to the [KIC project in Red Hat partners portal](https://connect.redhat.com/projects/5eab1b8ea165648bd1353266/images) to confirm that E2E tests are succeeding. If you are backporting features into a non-main branch, run a [targeted E2E job against that branch](https://github.com/Kong/kubernetes-ingress-controller/actions/workflows/e2e_targeted.yaml).
+      - label: Click publish button for all tags to publish the image to Red Hat Catalog.
```

Comment:
For all or for the just pushed tag that for the release in question?

Reply:
We all pushed tags. For each pushed tag there is a publish button.

Reply:
What I'm trying to say it that when you release vA.B.C you're concerned to publish that, not any other version that's left unpublished (this shouldn't really happen, right?)

Reply:
But we also publish vA.B and latest.

Reply:
Change to be more verbose.